sections/featured-product.liquid

```diff
@@ -807,67 +807,39 @@
       "name": "t:sections.featured-product.blocks.variant_picker.name",
       "limit": 1,
       "settings": [
-        {
-          "type": "header",
-          "content": "t:sections.featured-product.blocks.variant_picker.settings.options.header"
-        },
         {
           "type": "select",
           "id": "picker_type",
           "options": [
             {
               "value": "dropdown",
-              "label": "t:sections.featured-product.blocks.variant_picker.settings.options.picker_type.options__1.label"
+              "label": "t:sections.featured-product.blocks.variant_picker.settings.picker_type.options__1.label"
             },
             {
               "value": "button",
-              "label": "t:sections.featured-product.blocks.variant_picker.settings.options.picker_type.options__2.label"
+              "label": "t:sections.featured-product.blocks.variant_picker.settings.picker_type.options__2.label"
             }
           ],
           "default": "button",
-          "label": "t:sections.featured-product.blocks.variant_picker.settings.options.picker_type.label"
-        },
-        {
-          "type": "header",
-          "content": "t:sections.featured-product.blocks.variant_picker.settings.options_with_swatches.header",
-          "info": "t:sections.featured-product.blocks.variant_picker.settings.options_with_swatches.info"
-        },
-        {
-          "id": "swatch_picker_type",
-          "type": "select",
-          "label": "t:sections.featured-product.blocks.variant_picker.settings.options_with_swatches.picker_type.label",
-          "options": [
-            {
-              "value": "swatch",
-              "label": "t:sections.featured-product.blocks.variant_picker.settings.options_with_swatches.picker_type.options__1.label"
-            },
-            {
-              "value": "swatch_dropdown",
-              "label": "t:sections.featured-product.blocks.variant_picker.settings.options_with_swatches.picker_type.options__2.label"
-            },
-            {
-              "value": "dropdown",
-              "label": "t:sections.featured-product.blocks.variant_picker.settings.options_with_swatches.picker_type.options__3.label"
-            },
-            {
-              "value": "button",
-              "label": "t:sections.featured-product.blocks.variant_picker.settings.options_with_swatches.picker_type.options__4.label"
-            }
-          ],
-          "default": "swatch"
+          "label": "t:sections.featured-product.blocks.variant_picker.settings.picker_type.label"
         },
         {
           "id": "swatch_shape",
```

Comment:
Should we change the id of this setting to `swatch` to align with the new name/purpose of the setting?

Reply:
Lets keep it as is for now but if we are able to manually change this once we get translations I think we should